### PR TITLE
fix(upsmon): wrong dict

### PR DESCRIPTION
**Pull request by martintamare**
Something was wrong: config key was shared between filename and context.

### CODE REVIEW

**Comment by martintamare** [comment]
Also in upsmon, configuration was reading type, an,d thus defaulting to default 'master'. Now take key type from dict.

File: nut/server.sls [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {% from "nut/map.jinja" import nut with context %}
 {% set ups_config = nut.server.ups if nut.server.ups is defined else {} %}
-{% set upsd_config = nut.server.upsd.config if nut.server.upsd.config is defined else {} %}
+{% set upsd_config = nut.server.upsd.upsd if nut.server.upsd.upsd is defined else {} %}
```
Comment (javierbertoli) [inline nut/server.sls]: @martintamare, you're right it wrong, but it should actually be

```yaml
{% set upsd_config = nut.server.upsd if nut.server.upsd is defined else {} %}
```
to match the logic of `ups_config`. `upsd_users` is the 'special case' here.

**Comment by martintamare** [comment]
Thanks for the review, patch applied !

**Comment by javierbertoli** [comment]
Merged, thanks for fixing this, @martintamare !

**Pull request merged**